recording_server/src/utils/Logger.ts

```
@@ -111,160 +108,149 @@ export function setupConsoleLogger() {
 
         switch (options.type) {
             case 'normal':
             case 'force-termination':
                 if (!options.bot_uuid || !options.secret) {
-                    throw new Error(
-                        'bot_uuid and secret are required for normal log upload',
-                    )
+                    throw new Error('bot_uuid and secret are required for normal log upload');
+                }
+                const pathManager = PathManager.getInstance(options.bot_uuid, options.secret);
+                logPath = currentBotLogFile || pathManager.getLogPath();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)